content/v2.0/reference/flux/stdlib/influxdb-v1/fieldkeys.md

```diff
@@ -0,0 +1,67 @@
+---
+title: v1.fieldKeys() function
+description: The `v1.fieldKeys()` function returns a list of fields in a bucket.
+menu:
+  v2_0_ref:
+    name: v1.fieldKeys
+    parent: InfluxDB v1
+weight: 301
+v2.0/tags: [fields]
+related:
+  - /v2.0/query-data/flux/explore-schema/
+  - https://docs.influxdata.com/influxdb/latest/query_language/schema_exploration#show-field-keys, SHOW FIELD KEYS in InfluxQL
+---
+
+The `v1.fieldKeys()` function returns a list of fields in a bucket.
+The return value is always a single table with a single column, `_value`.
+
+```js
+import "influxdata/influxdb/v1"
+
+v1.fieldKeys(
+  bucket: "example-bucket",
+  predicate: (r) => true,
```

Comment:
what happens if predicate isn't included?

Reply:
The default is `(r) => true`, which means all values are returned. Nothing gets filtered out.